### PR TITLE
AAD rename for all content owned by wp

### LINKED ISSUE #30592: AAD rename for all tutorials and other content owned by WP
Contributes to #30185

This is to track AAD rename updates in all topics I own outside of gRPC and SignalR (which are being tracked seperately) such as tutorials, etc.

Replace "Azure Active Directory," "Azure AD," and "AAD"

List topics here:

None for the tutorials node, but there are tutorials that use the following includes that require the rename:
- [ aspnetcore/includes/DuendeldentityServer ] 

Examples:

Product references:

Azure Active Directory
becomes
Microsoft Entra ID

Azure Active Directory (AAD)
becomes
Microsoft Entra ID (ME-ID), but only if we are really referring to "ME-ID" later, otherwise it would be "Microsoft Entra ID".

Azure AD
becomes
Microsoft Entra ID

AAD
becomes
ME-ID

Topic file names and their links:
azure-active-directory.md
might become
microsoft-entra-id

Links:
Azure Active Directory (AAD) or Azure Active Directory
becomes
Microsoft Entra ID (ME-ID)  or just "Microsoft Entra ID" if there was no need to refer to later ME-ID after first use.

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #30592

Searched through entire aspnetcore\tutorials node and for anything in aspnetcore/ that has wadepickett in the metadata.

Topics were clean (beyond the gRPC and SignalR nodes which are taken care of in other PR's)
However, I had topics that used includes that needed renaming, so that is what I am updating in this PR.

Found and fixed AAD references here:
aspnetcore/includes/DuendeldentityServer.md  (used in the MongoDB tutorial for example).
aspnetcore/includes/IdentityServer4.md

Checked for:
"Azure Active"
"Azure AD"
"Azure-active"
"Azure-ad"
"AAD" (which brings up 20 random doc ID's that had "aad" in the sequence but no legit AAD hits).

### CODE REVIEW

**Comment by wadepickett** [comment]
Last one.  Sorry to have split these renames between 3 PR's.  I thought originally, I would find more hits per those nodes.

File: aspnetcore/includes/DuendeIdentityServer.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ASP.NET Core Identity adds user interface (UI) login functionality to ASP.NET Core web apps. To secure web APIs and SPAs, use one of the following:
 
-* [Azure Active Directory](/azure/api-management/api-management-howto-protect-backend-with-aad)
-* [Azure Active Directory B2C](/azure/active-directory-b2c/active-directory-b2c-custom-rest-api-netfw) (Azure AD B2C)
+* [Microsoft Entra ID](/azure/api-management/api-management-howto-protect-backend-with-aad)
+* [Microsoft Entra ID B2C](/azure/active-directory-b2c/active-directory-b2c-custom-rest-api-netfw)
```
Comment (tdykstra) [inline aspnetcore/includes/DuendeIdentityServer.md:4]: I think Azure AD B2C is supposed to be left unchanged.

File: aspnetcore/includes/IdentityServer4.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ASP.NET Core Identity adds user interface (UI) login functionality to ASP.NET Core web apps. To secure web APIs and SPAs, use one of the following:
 
-* [Azure Active Directory](/azure/api-management/api-management-howto-protect-backend-with-aad)
-* [Azure Active Directory B2C](/azure/active-directory-b2c/active-directory-b2c-custom-rest-api-netfw) (Azure AD B2C)
+* [Microsoft Entra ID](/azure/api-management/api-management-howto-protect-backend-with-aad)
+* [Microsoft Entra ID B2C](/azure/active-directory-b2c/active-directory-b2c-custom-rest-api-netfw)
```
Comment (tdykstra) [inline aspnetcore/includes/IdentityServer4.md:4]: revert

**Review by tdykstra — APPROVED** [review]

File: aspnetcore/includes/DuendeIdentityServer.md (hunk 1 above)
Comment (wadepickett) [inline aspnetcore/includes/DuendeIdentityServer.md:4]:
```suggestion
* [Azure Active Directory B2C](/azure/active-directory-b2c/active-directory-b2c-custom-rest-api-netfw) (Azure AD B2C)
```

File: aspnetcore/includes/IdentityServer4.md (hunk 2 above)
Comment (wadepickett) [inline aspnetcore/includes/IdentityServer4.md:4]:
```suggestion
* [Azure Active Directory B2C](/azure/active-directory-b2c/active-directory-b2c-custom-rest-api-netfw) (Azure AD B2C)
```

Comment (wadepickett) [reply aspnetcore/includes/IdentityServer4.md]: Thanks!  Reverted.  Looked it back up, you are right,  B2B changed but not B2C.

**Pull request merged**